shared/build.gradle.kts

```diff
@@ -87,6 +88,14 @@ android {
     namespace = "template.shared"
 }
 
+sqldelight {
+    databases {
+        create("AppDatabase") {
+            packageName.set("template.shared")
+        }
+    }
+}
```

Comment:
We meed to solve for:

1. How to remove this entire block if the user does not want sqldelight dependency (could move into another file?)
2. Update the packageName if we do keep it.